vagga.yaml

```diff
@@ -0,0 +1,26 @@
+containers:
+  pip-freeze:
+    setup:
+      - !Alpine v3.4
+      - !Py3Install
+        - pip
+        - 'Django >= 1.9,<=1.10.5'
+        - 'gunicorn >=19.5,<=16.6.0'
+        - 'whitenoise >= 3.2,<=3.2.3'
+        - 'django-environ >= 0.4,<=0.4.1'
```

Comment:
Hmm...
But what if in next realize will be something that broke my system? (comment a bit higher)
I don't really know all the features in each version